### ISSUE TITLE
Binding.pry detector

### ISSUE THREAD

**Issue by roman-dubrovsky** [datarockets/datarockets-style#139]
**Is your feature request related to a problem? Please describe.**

The cop which is finding `binding.pry` in the code

**Comment by lazycoder9** [comment]
@roman-dubrovsky I see that our config is already detects `binding.pry` and `Kernel.binding.pry`, but it doesn't check `binding.irb`
![image](https://user-images.githubusercontent.com/19951609/71721062-92e00f00-2e45-11ea-912c-fcc10895b22c.png)

**Comment by nikitasakov** [comment]
The same situation for me ^. Will investigate how to find all bindings

**Comment by nikitasakov** [comment]
I was investigating this issue and it is result:

`binding.irb` is a feature of ruby >= 2.4 (https://blog.bigbinary.com/2017/04/18/binding-irb.html)

We have this in our `.rubocop.yml`:
```
AllCops:
  TargetRubyVersion: 2.3
```

So it doesn't detect this `binding.irb` because ruby 2.3 doesn't support `binding.irb`.

If I change `TargetRubyVersion` to 2.6, rubocop is able to see it
![image](https://user-images.githubusercontent.com/32716983/71882627-e011f880-3145-11ea-8ea7-4cb3b242a9d9.png)

So looks like our rubocop is able to find `binding`s.

Is this issue relevant?

**Comment by AleksSenkou** [comment]
@nikitasakov is it checking `binding.pry` for ruby 2.3?

**Comment by nikitasakov** [comment]
@AleksSenkou Yes

**Comment by AleksSenkou** [comment]
@nikitasakov is something stopping us from closing this issue then?

**Comment by nikitasakov** [comment]
@AleksSenkou For me - no, it is just a bit unclear for me when discussion is finished. Because of it I don't understand when we can close issues. 🤷‍♂️For some reason this issue is created and maybe we missed something

**Comment by AleksSenkou** [comment]
@nikitasakov I understand your thoughts, man

Time and energy saver is to close an issue if you think that we can point dot on the conversation. If it's still important for someone - they will reopen the issue and start discussion once again 😉

**Comment by nikitasakov** [comment]
@AleksSenkou Got it, thanks 🙂
